homeassistant/components/media_player/xiaomi_tv.py

```diff
@@ -0,0 +1,112 @@
+"""
+Add support for the Xiaomi TVs.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/xiaomi_tv/
```

Comment:
The url is wrong. Look at an existing platform to see the correct format.

Reply:
Right yes, I had absolutely missed that. The correct url would be `https://home-assistant.io/components/media_player.xiaomi_tv/`.